### PR TITLE
207 / rent modal property

### PR DESCRIPTION
**Pull request by mishaadzh**
After approving USDT I try to rent and get an error in metamask  "This transaction is expected to fail". I got the same mistake when I rented for ETH and the rental price was very small

### CODE REVIEW

**Comment by mishaadzh** [comment]
> You can prompt MetaMask to execute the transaction anyways. Can you do that, so we can figure out what exactly is the mistake?



> 

Error: Transaction has been reverted by the EVM
image

**Comment by failfmi** [comment]
You do not have USDC tokens. I've sent you some.

**Comment by mishaadzh** [comment]
> 


Yes, I had a little less :)

**Comment by failfmi** [comment]
image
There are a few properties that the datepicker cannot be changed, e.g.:
* /property/39 end date is not preselected
* /property/17 end date cannot be changed

**Comment by failfmi** [comment]
image
the styling is broken here as well

**Comment by mishaadzh** [comment]
> property/19 - testnet. If you select a previous time it does not show the error message.

is it due to the "paymentToken? .id! == ONE_ADDRESS" condition, I don't know if it's relevant?

**Comment by mishaadzh** [comment]
> property/19 - testnet. If you select a previous time it does not show the error message.

now it works with the condition "payment Token? .id! == ONE ADDRESS", ie only with USDC. This condition already existed, how can we change it?

File: src/modules/land-works/components/lands-rent-date-picker/index.tsx [hunk 1]
```diff
@@ -1,97 +1,111 @@
...
 
-      // Disable the hours only if its the same hour
-      if (current?.hour() === maxEndDate.hour()) {
-        // added one minute each to be able to choose  if MIN & MAX are same
-        const dMins = range(0, 60).filter((m) => m > maxEndDate.minute() + 1);
-        disabledMins = [...disabledMins, ...dMins];
-      }
+  useEffect(() => {
+    if (isMinEqualMax()) {
+      handleRentDateChange([minStartDate, maxEndDate.add(60, 'seconds')]);
```
Comment (failfmi) [inline src/modules/land-works/components/lands-rent-date-picker/index.tsx:46]: why do you add 60 seconds here?

Comment (mishaadzh) [reply src/modules/land-works/components/lands-rent-date-picker/index.tsx:46]: If at least 10 minutes, then in fact I got the difference between minStartDate and minRentPeriod 9 minutes. However, I will try to solve it

Comment (mishaadzh) [reply src/modules/land-works/components/lands-rent-date-picker/index.tsx]: > why do you add 60 seconds here?

done, for some reason one minute was added for the start date

**Review by failfmi — APPROVED** [review]

**Pull request merged**